torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -1,44 +1,100 @@
 defmodule Torchx.NxDoctestTest do
+  @moduledoc """
+
+  # Why some doctests are excluded?
+
+  ## Not callbacks
+
+  * default_backend/2
+
+  ## Python rounds differently, than Elixir (for now)
+
+  * round/1
+
+  ## Result mismatch in less significant digits (precision issue)
+
+  * logistic/1
+
+  ## Callback or test requires explicity backend transfer to BinaryBackend
+
+  * to_binary/2
+  * to_flat_list/2
+  * random_uniform/4
+  * random_normal/4
```

Comment:
Those should work now with the default backend!

Reply:
They do, but tests fail, because they use `Nx.to_binary/1` to check the result.

Reply:
I see, thanks!